samples/wasm/README.md

```diff
@@ -19,10 +19,10 @@ Check [AKS Edge Essentials requirements and support matrix](https://learn.micros
 1. Move to an appropriate working directory
 1. Download [Set-AksEdgeWasmRuntime.ps1](./Set-AksEdgeWasmRuntimes.ps1)
     ```powershell
-    Invoke-WebRequest -Uri "https://raw.githubusercontent.com/Azure/AKS-Edge/preview/samples/wasm/Set-AksEdgeWasmRuntimes.ps1" -OutFile ".\Set-AksEdgeWasmRuntimes.ps1"
+    Invoke-WebRequest -Uri "https://raw.githubusercontent.com/Azure/AKS-Edge/preview/samples/wasm/Set-AksEdgeWasmRuntime.ps1" -OutFile ".\Set-AksEdgeWasmRuntimes.ps1"
     Unblock-File -Path ".\Set-AksEdgeWasmRuntimes.ps1"
```

Comment:
good to fix the OutFile name also to be same. can be confusing later. 
-OutFile .\SetAksEdgeWasmRuntime.ps1 , same for unblock-file line.
You may also add a Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope  Process -Force

Reply:
Thanks - Good catch, changing it